zoomx/src/main/java/com/zoomx/zoomx/ui/request/RequestActivity.java

```diff
@@ -73,6 +80,12 @@ public boolean onCreateOptionsMenu(Menu menu) {
                 searchManager.getSearchableInfo(getComponentName()));
         searchView.setSubmitButtonEnabled(true);
```

Comment:
Can you transfer the search view code related to separated function ?

Reply:
Sure

Reply:
Done